### PR TITLE
Fix stats conv

### PR DESCRIPTION
**Pull request by llyyr**
Read this before you submit this pull request:
https://github.com/mpv-player/mpv/blob/master/DOCS/contribute.md

Reading this link and following the rules will get your pull request reviewed
and merged faster. Nobody wants lazy pull requests.

### CODE REVIEW

File: TOOLS/stats-conv.py [hunk 1]
```diff
@@ -161,6 +165,6 @@ def mkColor(t):
     else:
         args['pen'] = pg.mkPen(color, width=0)
     G.curveno[cur] += 1
-    n = cur.plot([x for x,y in e.vals], [y for x,y in e.vals], **args)
+    cur.plot([x for x, _ in e.vals], [y for _, y in e.vals], **args)
```
Comment (kasper93) [inline TOOLS/stats-conv.py:168]: maybe
```suggestion
    x_vals, y_vals = zip(*e.vals)
    cur.plot(x_vals, y_vals, **args)
```

Comment (llyyr) [reply TOOLS/stats-conv.py]: Done

**Comment by kasper93** [comment]
Events like `mistimed` are not longer working (being drawn) after those changes.

**Review by kasper93 — CHANGES_REQUESTED** [review]
Events like mistimed are not longer working (being drawn) after those changes.

**Comment by llyyr** [comment]
> Events like `mistimed` are not longer working (being drawn) after those changes.

`get_event(name, "event-signal")` and `get_event(event, "event-signal")` looked exactly the same to me 2 days ago...

Fixed.

**Review by kasper93 — APPROVED** [review]
works

**Pull request merged**